spec/workers/virus_scan_worker_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe VirusScanWorker, type: :worker do
+  let(:worker) { described_class.new }
```

Comment:
Very minor: I might be wrong, but I think you might be able to achieve this more succinctly with `subject(:worker)`.

Reply:
Rspec supports [implicit subject](https://relishapp.com/rspec/rspec-core/v/2-0/docs/subject/implicit-subject) but using them to make this more succinct violates [RuboCop::Cop::RSpec::NamedSubject](http://www.rubydoc.info/gems/rubocop-rspec/1.6.0/RuboCop/Cop/RSpec/NamedSubject). There's not much difference between `let(:worker) { described_class.new }` and `subject(:worker) { described_class.new }` so I'll leave it.